app.js

```diff
@@ -1,3 +1,65 @@
-/**
- * Created by Dima on 10.09.2016.
- */
+const http = require('http');
+const coldFilmParsers = require('./coldFilmParsers');
+const utf8 = require('utf8');
+const co = require('co');
+
+function getRequest(path,parser) {
+    return new Promise((resolve, reject) => {
+        let options = {
+            hostname: 'coldfilm.ru',
+            port: 80,
+            method: 'GET',
+            path
+        };
+        let request = http.request(options, (res) => {
+            let body = [];
+            res
+                .on('data', (chunk) => {
+                    body.push(chunk);
+                })
+                .on('end', () => {
+                    body = Buffer.concat(body).toString();
+                    let data = parser(body);
+                    resolve(data);
+                })
+        });
+
+        request.on('error', (err) => {
```

Comment:
You can pass `reject` as collback here
`request.on('error', reject);`
also if arrow function's body have only one line it's better to write just `err => reject(err)`
https://github.com/airbnb/javascript#arrow-functions